lib/Formatter.php

```diff
@@ -75,4 +97,16 @@ public function humanFileSize($bytes) {
 		return "$bytes PB";
 	}
 
+	public function getPlatformFromUA(string $userAgent) {
+		if ($userAgent === '') {
+			return 'None';
+		}
+		foreach ($this->userAgents as $platform => $regex) {
+			if (preg_match($regex, $userAgent)) {
+				return $platform;
```

Comment:
platform cannot be translated this way

Reply:
Should it though? I guess the occ output is supposed to be used for an outside api or tool?

Reply:
ok, i thought it would be presented to the end user as is. then, never mind